Jenkinsfile

```diff
@@ -368,6 +374,26 @@ pipeline {
   }
 }
 
+def uploadToCDN() {
+  def source = 'packages/rum/dist/bundles/*.js'
+  def target = 'gs://beats-ci-temp/rum' // TODO: change bucket
+  def secret = 'secret/observability-team/ci/service-account/test-google-storage-plugin' // TODO: change secret
+  def version = sh(label: 'Get package version', script: 'jq --raw-output .version packages/rum/package.json', returnStdout: true)
+  def majorVersion = "${version.split('\\.')[0]}.x"
```

Comment:
Version with the semver format: `<major>.<minor>.<patch>`

Major version with the format: `<major>.x`

__NOTE__: lowercases